Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.6-slim
```

Comment:
As far as I understand for Meltdown only the host OS matters since Docker uses the host kernel. I'm not sure about Spectre, but we also don't really have any secrets in the app process that aren't already available in the environment variables, so I'm not sure what our threat model is in this case.